pkg/volume/flexvolume/driver-call.go

```diff
@@ -235,3 +241,29 @@ func handleCmdResponse(cmd string, output []byte) (*DriverStatus, error) {
 
 	return &status, nil
 }
+
+// getDriverCapabilities returns the reported capabilities as returned by driver's init() function
+func getDriverCapabilities(plugin *flexVolumePlugin) (*driverCapabilities, error) {
+	call := plugin.NewDriverCall(initCmd)
```

Comment:
I have refactored so that init() is only called once:  https://github.com/kubernetes/kubernetes/pull/50843/commits/47b85573c9a53308a3273528a178df48b552676c